ci-operator/step-registry/servicemesh/istio/e2e/servicemesh-istio-e2e-commands.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+function check_pod_status() {
+    INTERVAL=60
+    CNT=10
+    while [ $((CNT)) -gt 0 ]; do
+        READY=false
+        while read -r i
+        do
+            pod_name=$(echo "${i}" | awk '{print $1}')
+            pod_phase=$(echo "${i}" | awk '{print $3}')
+            if [[ "${pod_phase}" == "Running" ]]; then
+                READY=true
+            else
+                echo "Waiting for Pod ${pod_name} to be ready"
+                READY=false
+            fi
+        done <<< "$(oc -n "${MAISTRA_NAMESPACE}" get pods "$1" --no-headers)"
+
+        if [[ "${READY}" == "true" ]]; then
+            echo "Pod $1 has successfully been deployed"
+            return 0
+        else
+            sleep "${INTERVAL}"
+            CNT=$((CNT))-1
+        fi
+
+        if [[ $((CNT)) -eq 0 ]]; then
+            echo "Pod $1 did not successfully deploy"
+            oc -n "${MAISTRA_NAMESPACE}" get pods "$1"
+            return 1
+        fi
+    done
+}
+
+
+MAISTRA_NAMESPACE=$(
```

Comment:
Why redefining this variable to its same value?

Reply:
Just for waiting the oc create command finish its ns creation. I can remove that way and put it into a function.